sdk/core/azure-core/CHANGELOG.md

```diff
@@ -14,425 +16,425 @@
 
 ### Features Added
 
-- When a `RequestFailedException` exception is thrown, the `what()` method now includes information about the HTTP request which failed.
-- Adding option `WinHttpTransportOptions.IgnoreUnknownCertificateAuthority`. It can be used to disable verifying server certificate for the `WinHttpTransport`.
+* When a `RequestFailedException` exception is thrown, the `what()` method now includes information about the HTTP request which failed.
```

Comment:
At this point, I'm wondering if it's just worth accepting the "-" to "*" change just to prevent these from happening again and again.

Reply:
I need to get rid of some VSCode extension that is adding this format after saving :)